### PR TITLE
Generate readmes for MCR portal

### PR DESCRIPTION
**Pull request by mthalman**
* Updates the manifest to conform to the new schema defined by https://github.com/dotnet/docker-tools/pull/1001.
* Defines new readme files to be used for the MCR portal.
* Refactors the readme templates to enable more common content between the GH and MCR readmes.

### CODE REVIEW

File: README.aspnet.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
-The images from the dotnet/nightly repositories include last-known-good (LKG) builds for the next release of [.NET](https://github.com/dotnet/core).
```
Comment (MichaelSimons) [inline README.aspnet.md:1]: Why move this from the top.  This is very important to highlight upfront for the nightly repos.  We don't want new folks to miss this callout.

File: manifest.json [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {
-  "readme": "README.md",
-  "readmeTemplate": "eng/readme-templates/README.md",
+  "readme": {
```
Comment (MichaelSimons) [inline manifest.json:2]: Did MCR explicitly leave out product family repos?  It feels like there is an opportunity here to provide a better experience for grouping related repos.

Comment (mthalman) [reply manifest.json:2]: Yes, they're not supported yet.

Comment (MichaelSimons) [reply manifest.json:2]: So it is something on their backlog?

Comment (mthalman) [reply manifest.json:2]: Yes

File: README.aspnet.md (hunk 1 above)
Comment (MichaelSimons) [reply README.aspnet.md:1]: My concern with Docker Hub is that the Featured Tags section is first.  My preference is to have this before the Featured Tags on DH.  On MCR the About section is first so it isn't an issue.  The bold Important does stand out and help though.

**Review by MichaelSimons — APPROVED** [review]

File: .mcr/portal/README.aspnet.portal.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+.NET Framework:
+
+* [dotnet/framework](https://hub.docker.com/_/microsoft-dotnet-framework/): .NET Framework, ASP.NET and WCF
+* [dotnet/framework/samples](https://hub.docker.com/_/microsoft-dotnet-framework-samples/): .NET Framework, ASP.NET and WCF Samples
+
+## Usage
+
+The [.NET Docker samples](https://github.com/dotnet/dotnet-docker/blob/main/samples/README.md) show various ways to use .NET and Docker together. See [Building Docker Images for .NET Applications](https://docs.microsoft.com/dotnet/core/docker/building-net-docker-images) to learn more.
+
+## Container sample: Run a web application
```
Comment (MichaelSimons) [inline .mcr/portal/README.aspnet.portal.md:39]: Should this be ###?

Comment (mthalman) [reply .mcr/portal/README.aspnet.portal.md]: Good find. Fixed

**Pull request merged**